packages/terra-aggregator/examples/tmp/DisclosureManager.jsx

```diff
@@ -0,0 +1,316 @@
+import React from 'react';
```

Comment:
This file will be moved into its own package shortly. Feel free to review if you want a preview.